packages/core-v2/src/entity-model/entity-model.ts

```diff
@@ -21,5 +21,9 @@ export interface EntityModel {
      * @returns Function that can be called to unsubscribe from the changes.
      */
     subscribeToChanges(callback: (updated: Record<string, Entity>, removed: string[]) => void): () => void;
-}
 
+    /**
+     * Returns Id of the model
+     */
+    getId(): string;
+}
```

Comment:
Entity modelu jsem pridal jeste idcko. Me se to hodi v dscme, abych se na ty modely jednoduse mohl odkazat. 
Moznost by jeste byla oznackovat si je jen u sebe, ale mam pocit, ze by se tohle mohlo hodit